public/js/user.js

```diff
@@ -95,16 +96,15 @@ apos.define('apostrophe-i18n-debugger', {
       }
 
       var walker = document.createTreeWalker(
-        document.body, 
-        NodeFilter.SHOW_TEXT, 
-        null, 
+        document.body,
+        NodeFilter.SHOW_TEXT,
+        null,
         false
       );
 
       var textNode;
-      var textNodes = [];
 
-      while (textNode = walker.nextNode()) {
+      while (textNode === walker.nextNode()) {
```

Comment:
Really not sure about this one.

Reply:
This breaks the code.

I think double paren is used as a convention to mean "I really meant to assign in the conditional of a while." The linter might honor that.

Reply:
Actually assigning in the conditional of a while is so common that this is... just annoying. I guess if we could use `for...of` (iterators) there would be a nice alternative but we can't.

Reply:
Give double paren a try?

Reply:
Mikey likes it.